packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -158,7 +168,23 @@ class PaginatedDataTable extends StatefulWidget {
   ///
   /// This value is optional and defaults to kMinInteractiveDimension if not
   /// specified.
-  final double dataRowHeight;
+  @Deprecated(
+    'Migrate to use dataRowMinHeight and dataRowMaxHeight instead. '
+    'This feature was deprecated after v3.5.0-10.0.pre.',
+  )
+  double? get dataRowHeight => dataRowMinHeight == dataRowMaxHeight ? dataRowMinHeight : null;
```

Comment:
Why is this changed to a getter, while in DataTable and DataTableTheme, there is no change?

Reply:
When adding the assertions for `PaginatedDataTable` it seemed clear to me that a separate field for `dataRowHeight` was not needed. For `DataTable` it was not so clear because `dataRowHeight` was used in the `_buildDataCell` method. However, thanks to your comment, I tried to make it consistent and it worked.
Please check the added commit.